app/assets/javascripts/dashboards.js

```diff
@@ -0,0 +1,5 @@
+$(function () {
+  $('#no-deploy-toggler').click(function(e) {
+    $('tr.no-elements').toggleClass('hidden');
```

Comment:
I tried it with `.toggle()`, but it didn't work for some reason.  Good call on making it more reusable though.  I'll do that.